### PR TITLE
ip-table-update

### PR DESCRIPTION
**Pull request by alchemyDocs**
There are updates to the IP tables. Review the latest changes.

### CODE REVIEW

**Review by bradbehle — CHANGES_REQUESTED** [review]
We need to do more testing of these changes, and probably have some of the squad leads meet to sign off on these, before they are merged

**Review by derekpoindexter — APPROVED** [review]

**Comment by derekpoindexter** [comment]
@bradbehle - can you take a look at this one again - i updated the script to pull only microservice clusters for the "control-plane-ip" files

**Comment by bradbehle** [comment]
These look right as far as the general size of the lists and number of subnets.  Also, I do think this is right to put only the microservices tugboat/control plane subnets in the control-plane-ips based on the recent doc clarifications we made (or are in the process of making), and put the full control plane IP subnet lists in the iam-firewall-ips lists.

Unfortunately, I don't have a good way to verify the control plane subnets.  Tomorrow I should spot check a few regions to make sure they look right

File: control-plane-ips/control-plane-ips-dal.txt [hunk 1]
```diff
@@ -1,44 +1,6 @@
...
-169.62.216.0/25
-169.62.222.0/25
-169.62.253.0/25
-169.63.18.128/25
-169.63.20.0/25
-169.63.24.0/24
-169.63.47.96/27
-169.63.199.128/25
-169.63.205.0/25
+52.116.235.224/28
```
Comment (bradbehle) [inline control-plane-ips/control-plane-ips-dal.txt:6]: Verified this

File: control-plane-ips/control-plane-ips-fra.txt [hunk 2]
```diff
@@ -1,52 +1,6 @@
...
-169.50.58.160/27
-169.50.177.128/25
-169.50.185.32/27
-169.51.3.64/26
-169.51.22.64/26
-169.51.28.128/25
-169.51.39.64/26
-169.51.41.64/26
-169.51.161.128/25
-169.51.193.0/24
```
Comment (bradbehle) [inline control-plane-ips/control-plane-ips-fra.txt:52]: Verified this

File: control-plane-ips/control-plane-ips-lon.txt [hunk 3]
```diff
@@ -1,47 +1,6 @@
...
-158.176.112.0/26
-158.176.131.107
 158.176.137.144/28
-158.176.161.0/25
-159.8.171.0/26
-159.122.249.128/25
-169.50.199.64/26
-169.50.220.32/27
-169.50.221.0/25
+158.176.95.0/27
```
Comment (bradbehle) [inline control-plane-ips/control-plane-ips-lon.txt:6]: Verified this

File: control-plane-ips/control-plane-ips-osa.txt [hunk 4]
```diff
@@ -1,21 +1,6 @@
...
-163.69.66.32/27
 163.69.66.64/27
-163.69.66.96/27
-163.73.64.80/28
-163.73.64.243
 163.73.65.224/28
-163.73.66.0/28
-163.73.66.64/27
 163.73.66.96/27
-163.73.66.128/27
```
Comment (bradbehle) [inline control-plane-ips/control-plane-ips-osa.txt:21]: Verified this

File: control-plane-ips/control-plane-ips-sao.txt [hunk 5]
```diff
@@ -1,17 +1,6 @@
...
-163.109.65.208/28
-163.109.66.96/27
 163.109.66.224/27
-169.57.159.96/27
-169.57.162.144/28
-169.57.181.16/28
 169.57.187.0/27
-169.57.190.64/26
-169.57.192.128/25
 169.57.219.16/28
```
Comment (bradbehle) [inline control-plane-ips/control-plane-ips-sao.txt:6]: Verified this

File: control-plane-ips/control-plane-ips-syd.txt [hunk 6]
```diff
@@ -1,21 +1,3 @@
...
-135.90.73.0/26
-135.90.75.0/27
-135.90.77.68
-135.90.78.128/26
 135.90.83.160/27
-168.1.20.148
 168.1.56.160/27
-168.1.199.0/26
-168.1.209.192/26
-168.1.212.128/25
```
Comment (bradbehle) [inline control-plane-ips/control-plane-ips-syd.txt:21]: Verified this

File: control-plane-ips/control-plane-ips-tok.txt [hunk 7]
```diff
@@ -1,25 +1,6 @@
...
 165.192.89.96/27
-165.192.101.48/28
-165.192.105.64/27
-169.38.97.192/28
-169.38.111.192/26
-169.38.113.64/27
-169.56.40.128/25
 169.56.49.192/28
-169.56.110.64/26
-169.56.143.0/26
```
Comment (bradbehle) [inline control-plane-ips/control-plane-ips-tok.txt:25]: Verified this

File: control-plane-ips/control-plane-ips-tor.txt [hunk 8]
```diff
@@ -1,20 +1,6 @@
...
 163.74.69.0/27
-163.75.64.144/28
-163.75.64.176/28
-163.75.64.213
 163.75.65.240/28
-163.75.66.32/27
 163.75.67.192/27
-169.53.170.160/28
-169.53.178.192/26
-169.55.148.128/25
```
Comment (bradbehle) [inline control-plane-ips/control-plane-ips-tor.txt:20]: Verified this

File: control-plane-ips/control-plane-ips-wdc.txt [hunk 9]
```diff
@@ -1,51 +1,6 @@
...
-169.63.84.192/28
-169.63.89.112/28
-169.63.121.164
-169.63.137.0/25
-169.63.157.0/26
-169.63.163.160/27
-169.63.166.224/27
-169.63.170.60
-169.63.174.96/28
+52.117.100.64/27
```
Comment (bradbehle) [inline control-plane-ips/control-plane-ips-wdc.txt:6]: Verified this

**Review by bradbehle — APPROVED** [review]
I was able to find a way to verify the control-plane-ips, they are correct, so this looks good.  Thanks for generating these

**Pull request merged**